src/language/en-US.json

```diff
@@ -422,10 +428,10 @@
   "New_root_space": "New root space",
   "Subspace": "Sub-namespace",
   "Input_space_name": "Input namespace",
-  "namespace_tips_key_1": "1. The length of the root namespace is limited to 16 characters, the length of the secondary namespace is limited to 64 characters. Valid characters:",
-  "namespace_tips_key_2": "2. The first character of the namespace must use English letters. Values not allowed include:",
+  "namespace_tips_key_1": "1. The name must be unique in the network, and may have a maximum length of 64 characters. Valid characters:",
+  "namespace_tips_key_2": "2. Values not allowed include:",
```

Comment:
`2. Following characters are not accepted: `

Reply:
fix translation of the mosaic flags and namespace tips